crimeApi/index.js

```diff
@@ -0,0 +1,87 @@
+var http = require('http');
+
+//import express
+var express = require('express');
+
+//report Route to send images to the uploads folder
+var reportRoute = require('./route/report');
+
+//Image Route to send images to the uploads folder
+var imageRoute = require('./route/image')
+
```

Comment:
Please delete this duplicate line that's commented out

Reply:
Was this change implemented?